src/json/state_machine_input.json

```diff
@@ -119,6 +119,27 @@
       ]
     }
   },
+  "vpcpreconfig":{  
+    "credentials":null,
+    "federation":{  
+       "roles":[  
+
+       ],
+       "authorizer_user_guid":null,
+       "durationSeconds":0
+    },
+    "manageVpcPreConfig":{  
```

Comment:
Can we change it to "managedVpcPreConfig"

Reply:
I analyzed the impact of this change in state machine and i have to make changes in 16 places in different files where i am using this reference object.